### ISSUE TITLE
Difficulty Deploying Project with Docker Compose and Neo4j Aura DB

### ISSUE THREAD

**Issue by walmeidadf** [neo4j-labs/llm-graph-builder#412]
Hi everyone,

I'm not able to deploy my project using Docker Compose on my notebook and I´d like some support to do it.

I've tried two approaches. First I tried to ise a Neo4j database within the Docker Compose setup.
Then, now I'm utilizing an Aura DB instance on the cloud.

In both scenarios, I face the same problem. When attempting to log in on the frontend machine (http://localhost:8080), after providing connection information, the screen neither successfully connects to the database nor displays any error messages on either the screen or the Docker host.

I've attached several screenshots to illustrate the situation on the last scenario (using AuraDB):

Container Configuration: This screenshot showcases the configuration of both backend and frontend containers running on my machine, with Aura DB running in the cloud. I've used the export command to display the environment variables.
![backend-export](https://github.com/neo4j-labs/llm-graph-builder/assets/22039235/fc30a71b-7d32-4c79-85fb-e9128f0d669b)
![frontend-export](https://github.com/neo4j-labs/llm-graph-builder/assets/22039235/22ae7d93-e993-465f-936a-2c2b43c04565)

Frontend-Backend Communication: This screenshot demonstrates successful communication between the frontend and backend using a curl command.
![frontend-curl-backend](https://github.com/neo4j-labs/llm-graph-builder/assets/22039235/adb6f323-3c3b-4e14-8b05-10847788be17)

Backend API Connection: This screenshot shows my host machine successfully connecting to the backend's Swagger interface and executing the "connect" method with no errors.
![backend-swagger-connect](https://github.com/neo4j-labs/llm-graph-builder/assets/22039235/e732d39e-15b0-433c-86e8-e61e396ad715)

While I believe I've configured the environment variables correctly in both containers, the lack of error messages makes troubleshooting challenging.

I'd greatly appreciate any insights or suggestions you might have to help me resolve this connection problem.

Thank you!

**Comment by jexp** [comment]
Could you have a look here and see if that helps?

https://neo4j.com/labs/genai-ecosystem/llm-graph-builder-deployment/

**Comment by walmeidadf** [comment]
Hi @jexp,

Thanks for taking the time to provide feedback on my lab!

I believe I've found the issue. I initially misunderstood that the front-end container connected directly to the back-end container. This led me to change the BACKEND_API_URL parameter from "localhost:8000" to "backend:8000". Additionally, I performed the curl test from the front-end container based on this false assumption.

Following your feedback, I went through the lab again and identified the BACKEND_API_URL parameter as the only change I made. After switching it back to "localhost," the connection functioned as expected.
